### ISSUE TITLE
Please: Allow to disable warning messages for certain stations

### ISSUE THREAD

**Issue by leandor** [Yousei9/Logistic-Train-Network#332]
**TL;DR**
Could it be possible to add an option to disable the warning messages for given stations for when trains leave with cargo and also for when trains arrive with cargo to a depot? 

I mean these messages:

![image](https://github.com/0ptera/Logistic-Train-Network/assets/1095373/4b88cb34-5f4a-4cf3-b9e8-637e3e7eb597)

**What ?**
There's an use case for trains that provide certain things and upon return they come back with materials for recycling in a single trip. Case in point: Angel & Bob Red/Gree/Blue/Yellow Metal Catalyst vs the obtained Metal-catalysts carrier frame returned when they are consumed.

I.e. I have a station requesting the catalysts, and when the train arrives it's unloaded of its catalysts and loaded with the used carriers, and upon arriving back to the depot it's emptied so the carriers can be recycled.

I prefer to handle them this way, because I can make them all near the Ore sorting block (where the ores are located) and provided them to the network where they are needed with minimal complications (i.e. a single station), vs the other option which is providing the ores, since for that you'd need many more stations/requests since each catalysts consists of two metals (but you get rid of the recycling part, true.)  I.e. 4 items plus recycling vs 8 items and no recycling.

Granted, AFAIK this could be the only one such usage for this feature that I know of at the moment, but as long as the defaults remain as they are now, allowing players that need them to disable those spurious messages (in this specific case) wouldn't cause any problems for normal usage, I don't think.

**Why ?**
The constant messages under normal functioning gets annoying and hides other potential real issues coming from other stations that work normally, since other than the station names, there are no way to distinguish them at a glance.

Thanks and best wishes o/

**Comment by 0ptera** [comment]
Please read the Manual: https://forums.factorio.com/viewtopic.php?f=214&t=51072

What you're requesting exists for years in the form of a requester control signal "Disable Warnings".

**Comment by leandor** [comment]
Sorry, I tried that in the past, and at that time didn't seem to work in that particular base, so I completely forgot about mentioning that option this time.

I thought that that option only affected the 'No station supplying etc.'  message, and at least one of those warnings I mentioned don't generate at the requester, but instead on the depots when the train arrived there, so for that case didn't seem possible to me that setting that signal would have helped.

EDIT: I just verified in my current game, and it's just as I remembered: those warnings still come up even when the signal 'disable warnings' is set in the requester and the provider, haven't tried setting in the depots since I use the LTN Combinator mod that doesn't let me to enable that signal from its UI for depots.

I can upload a savegame google drive and post the link here if you'd want to check it.

**Comment by 0ptera** [comment]
Ah my bad, I missed the part where you specify it being the depot cargo warning.
Sry depots are unaffected by that signal.

I'll reopen it as enhancement.
However with Factorio 2.0 around the corner to completely replace LTN I don't feel particularly motivated to add features anymore.

**Comment by leandor** [comment]
Thank you! Yeah, I completely understand!

Still, there are probable some of us that run Factorio on old hardware that won't be able to run the new 2.0 engine version even without the expansion (I assume it will have updated hardware requirements, surely! With all the extra eye-candy they're seem to be adding...)

Even still, I mostly play modpacks/overhauls, so for my case at least, I won't probably update (assuming that I can run the update for a moment,) until the mods I like are ported/updated for the 2.0 version!

Not trying to convince you, of course! Besides, the issue I reported is mostly a super minor annoyance, that doesn't affect the use of the mod at all, to the point that this is my 4th A&B base where I use LTN, and I noticed this particular minor annoyance w/the warnings since the first time, but never cared enough to report it before! (The fact that I had lost access to my GH account for a time, could have had some part in that :) )

Best wishes!
